open-sauced/app
# fix: remove duplicate date range select

## Description
Remove the duplicate date range select from the `Activity(Pull request)` tab

fixes: #3762
## Related Tickets & Documents


## Mobile & Desktop Screenshots/Recordings
Before

<img width="385" alt="351129458-88cdc35f-8302-49c4-a78f-449d0db11ba5" src="https://github.com/user-attachments/assets/2c54b8a2-bf5a-4189-9c1f-476eb9764b5c">

After

![Screenshot from 2024-08-13 19-20-03](https://github.com/user-attachments/assets/fdff5022-1725-460a-b1b7-8cfe33223efd)





## Steps to QA

1. Go to the workspace in mobile mode
2. Notice, That no duplicate date range select in the Pull request Tab like Issues

## Tier (staff will fill in)


## [optional] What gif best describes this PR or how it makes you feel?
